common/source/docs/common-rpm.rst

```diff
@@ -66,38 +68,37 @@ and is recommended.
    pull-up resistor to be added between the Vcc and signal lines.  In some cases, their Vcc, gnd, and signal lines won't match the colour coding of the wire.  Hence,
    it is advisable to experiment with the provided Arduino sketch and a bread board to be sure of the sensor pin orientation.
 
-For a reliable RPM signal the hall effect pick-up should be mounted very close to the magnet.  The image below shows an example mounting on helicopter.
+For a reliable RPM signal the hall effect pick-up should be mounted very close to the magnet.  The image below shows an example mounting on a helicopter.
 
 .. image:: ../../../images/Installed_Hall_Effect_Sensor.jpg
     :target: ../_images/Installed_Hall_Effect_Sensor.jpg
     :width: 240px
 
-The three wires should then be plugged into autopilot.  This is commonly done using a standard servo plug.  Ground to Gnd, Vcc to 5V, and the signal line 
+The three wires should then be plugged into the autopilot.  This is commonly done using a standard servo plug.  Ground to Gnd, Vcc to 5V, and the signal line 
 needs to be attached to a GPIO pin.  On most smaller boards this will be any one of the PWM pins on the servo rail.  On a Pixhawk this must be one of the AUX ports.  
-For reasons explained later it is recommended to use the highest number pin first.  E.g. AUX 6 on a Pixhawk Cube or PWM 9 on an F405-Wing.
+For reasons explained later it is recommended to use the highest number pin first.  E.g. AUX 6 on a Pixhawk/Cube or PWM 9 on an F405-Wing.
 
 **Parameter Setup**
 
 First the board needs to be configured to allow PWM pins to be set for GPIO.  This is done using the parameter ``BRD_PWM_COUNT`` .  Reduce the PWM count to free up a pin to 
 be used for GPIO.  On non-Pixhawk boards the PWM count will include all PWM outputs.  On Pixhawk boards this parameter only affects AUX pins.  Write the parameter 
 and reboot the autopilot.
 
-Now the autopilot needs to be told which pin to find the RPM signal is on.  To do this you will need to find the pin number in the hwdef.dat file for you 
-board.  These can be found `here <https://github.com/ArduPilot/ardupilot/tree/master/libraries/AP_HAL_ChibiOS/hwdef>`__.  Within the file for your board you will 
-need to find where the GPIO(*) definition is.  The below example is an excerpt from the hwdef.dat file for the cube: ::
+Now the RPM library must be enabled. In the following sections, we will use the second instance of RPM sensor for parameter examples.
 
-    PD13 TIM4_CH2 TIM4 PWM(5) GPIO(54)
+Set the parameter :ref:`RPM2_TYPE<RPM2_TYPE>` to 1 for a GPIO pin based sensor.  Write the parameter to ArduPilot then refresh/fetch the 
```

Comment:
Can the example set RPM2_TYPE to 2 please.  We have left type = 1 active so as not to break any existing setups but the param meta-data does now say "1:Not Used" so we should maintain the theme in the wiki.

Reply:
done